.all-contributorsrc

```diff
@@ -247,6 +247,13 @@
       "contributions": [
         "bug"
       ]
+    },
+    {
+      "login": "Garbee",
+      "name": "Jonathan Garbee",
+      "avatar_url": "https://avatars.githubusercontent.com/u/868301?v=3",
+      "profile": "http://jonathan.garbee.me",
+      "contributions": []
```

Comment:
gah, I did. It failed and I didn't catch that looking at it. At least I think I selected something... I'll get that fixed.